src/main/resources/db/migration/V1_68__create_utilisation_report.sql

```diff
@@ -0,0 +1,307 @@
+-- Function to return daily utilisation stats for a court between two dates
+CREATE OR REPLACE FUNCTION juror_mod.util_report_daily_summary(p_loc_code text, p_start_date date, p_end_date date)
+ RETURNS TABLE(
+	report_date date,
+	working bigint,
+	sitting bigint,
+	attended bigint,
+	non_attendance bigint,
+	utilisation numeric)
+ LANGUAGE plpgsql
+AS $function$
+begin
+
+return query select
+	report_main.report_date,
```

Comment:
may need a column to show this is the date being reported on ... not the date the report is run

Reply:
> may need a column to show this is the date being reported on ... not the date the report is run

I think Craig might mean comment not column

Reply:
have added a comment